bundle/Command/DeleteOldCollectedInfoCommand.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Command;
+
+use DateInterval;
+use DateTimeImmutable;
+use Exception;
+use Netgen\InformationCollection\API\Service\InformationCollection;
+use Netgen\InformationCollection\API\Value;
+use Netgen\InformationCollection\API\Value\Filter\ContentId;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Command\HelpCommand;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+use function array_filter;
+use function array_map;
+use function array_unique;
+use function count;
+use function explode;
+use function implode;
+use function is_array;
+use function is_string;
+use function sprintf;
+
+use const PHP_INT_MAX;
+
+final class DeleteOldCollectedInfoCommand extends Command
+{
+    protected static $defaultName = 'nginfocollector:delete';
+
+    /**
+     * @var DateInterval
+     */
+    private $period;
+
+    /**
+     * @var InformationCollection
+     */
+    private $infoCollection;
+
+    public function __construct(InformationCollection $infoCollection)
+    {
+        $this->infoCollection = $infoCollection;
+        // Parent constructor call is mandatory for commands registered as services
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('nginfocollector:delete');
+        $this->setDescription('Deletes collected data that is older than specified period.');
+        $this->setHelp('This command allows you to delete data collected by this library that is older than specified period.');
+
+        $this->setDefinition(
+            new InputDefinition(
+                [
+                    new InputOption('content-id', 'c', InputOption::VALUE_REQUIRED, 'Content id.'),
+                    new InputOption('field-identifiers', 'f', InputOption::VALUE_REQUIRED, 'Field definition identifiers list.'),
+                    new InputOption('period', 'p', InputOption::VALUE_REQUIRED, 'Attributes older that this period will be deleted.'),
+                    new InputOption('all', 'a', InputOption::VALUE_NONE, 'Delete all fields.'),
+                    new InputOption('neglect', 'nn', InputOption::VALUE_NONE, 'Do not ask for confirmation.'),
```

Comment:
Is there a value in this option? I mean, this deletes the data, you most certainly do want confirmation.

Reply:
Fixed